editor/src/clj/editor/app_view.clj

```diff
@@ -511,9 +512,10 @@
       (set-pane-visible! scene pane-kw false))))
 
 (handler/defhandler :preferences :global
-  (run [workspace prefs]
+  (run [workspace prefs app-view]
     (prefs-dialog/open-prefs prefs)
-    (workspace/update-build-settings! workspace prefs)))
+    (workspace/update-build-settings! workspace prefs)
+    (ui/invalidate-menubar-item! ::file)))
```

Comment:
Disabling auto-load of external changes activates the **Load External Changes** menu item in the **File** menu. When auto-load of external changes is enabled, the menu item will not be visible.